tests/integration/components/login-form/component-test.js

```diff
@@ -0,0 +1,25 @@
+import { moduleForComponent, test } from 'ember-qunit';
```

Comment:
This test will not not pass `ember test`. 

This might be of some use @amberolive.
https://guides.emberjs.com/v2.11.0/testing/testing-components/